sites/public/pages/listings.tsx

```diff
@@ -114,6 +117,30 @@ const ListingsPage = () => {
               options={preferredUnitOptions}
               defaultValue={filterState?.bedrooms?.toString()}
             />
+            <Field
+              id="zipCodeField"
+              name={ListingFilterKeys.zipcode}
+              label={t("listingFilters.zipCode")}
+              register={register}
+              controlClassName="control"
+              placeholder={t("listingFilters.zipCodeDescription")}
+              validation={{
+                minLength: 5,
+                pattern: /[0-9 ,]*/,
```

Comment:
If we want to be stricter, maybe `[0-9]{5}(, [0-9]{5})*`?

Reply:
I played around with that pattern and others and realized that it wasn't working correctly because the first zip code would always match the first part of the regex, so even if the rest of the string didn't match, the resulting array would be ["11111", undefined] and this is "truthy". So I pulled this into a method that actually validates that each comma separated value is a valid 5 digit number.